manifests/configmap_infrastructure_roles.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: postgresql-infrastructure-roles
+data:
+  batman: |
+    name: batman
+    inrole: [role1, role2]
+    user_flags:
+      - superuser
+      - login
+    db_parameters:
```

Comment:
Since it's our example infra role manifest, could you please shortly comment inline on what `inrole, user_flags and db_parameters` mean ?

Reply:
I've added few commentaries for `inrole` and `db_parameters`, `user_flags` is self-explanatory I think.